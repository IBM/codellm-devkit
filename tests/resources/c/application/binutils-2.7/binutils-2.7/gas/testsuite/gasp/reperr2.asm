
	.AREPEAT	5
	.AENDR		
	.AENDR
	.END

