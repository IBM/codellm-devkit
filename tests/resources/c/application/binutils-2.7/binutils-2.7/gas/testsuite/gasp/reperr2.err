reperr2.asm:4 AENDR without a AREPEAT.
