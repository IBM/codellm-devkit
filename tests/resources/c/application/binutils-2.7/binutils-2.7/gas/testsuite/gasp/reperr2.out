!

!	.AREPEAT	5
!	.AENDR		
!	.AREPEAT	4
!	.AENDR
!	.AREPEAT	3
!	.AENDR
!	.AREPEAT	2
!	.AENDR
!	.AREPEAT	1
!	.AENDR
!	.AENDR
!	.END
